### DISCUSSION TITLE
Keyboard shortcut for opening search triggers without Shift when Caps Lock is on

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#30573] [Other Feature Feedback, Questions, & Ideas]
1. Open any file on Github, e.g. https://github.com/Instagram/LibCST/blob/main/libcst/_nodes/module.py
2. Turn Caps Lock on
3. Use Ctrl+F
4. See the problem.

Code Search should only get opened if Ctrl+Shift+F combination is actually used rather than be triggered whenever caps lock is on while you try using Ctrl+F to use the browser's find functionality.

**Comment by Akash1134** [comment]
Hi @jack1142 Thanks for sharing this Search behaviour with us. I have a couple of questions in order to reproduce the issue at my end.

- Which Web browser are you using?
- Are you working with the default keyboard/trackpad with your computer? 
- If you have another browser installed, does the issue persists? Did a recent update changed this behaviour that you might have noticed?

Thanks!

**Reply by Jackenmen** [reply]
> Which Web browser are you using?

Google Chrome 104 on Kubuntu 22.04.1 LTS

> Are you working with the default keyboard/trackpad with your computer?

I'm using a PC so the keyboard/mouse are separate devices. However, I can reproduce it in Google Chrome on my laptop that has Windows 10 on it and I'm just using the "default" keyboard/trackpad there.

> If you have another browser installed, does the issue persists?

Yes, it persists on Firefox 104 as well.

> Did a recent update changed this behaviour that you might have noticed?

No, I actually am pretty sure I ran into this same issue on April 28, 2022 (it annoyed me enough that I ranted to a friend in a chat about it so that's why I'm able to specify the exact date :)) *but* I couldn't figure out the cause of it, probably because I must have disabled Caps Lock shortly after and just not connect the two together. When that happened, I was using Windows 10 with Google Chrome on my PC.
